src/components/SearchOverlay/SearchOverlay.scss

```diff
@@ -147,10 +147,25 @@
       
       &.visible {
         display: flex;
+        flex-wrap: wrap;
         padding: 5px 0 0 20px;
       }
     }
 
+    .search-option {
+      width: 115px;
+    }
+
+    .wildcard {
+      display: none;
```

Comment:
Sorry I'm having a little trouble finding where this classname is used... Can you tell me which element is using this?

Reply:
Good catch, this was the class I was using when I had the wildcard option in its own div. Removed!